### ISSUE TITLE
QR Code option disabled...

### ISSUE THREAD

**Issue by ApacheThunder** [kwsch/PKHeX#753]
I can no longer generate QR codes. Is there a particular reason for this change or was it unintentional?

**Comment by kwsch** [comment]
Unable to replicate the issue described.

**Comment by evandixon** [comment]
Works for me, when using the default save
![image](https://cloud.githubusercontent.com/assets/2699524/22340043/b1c89608-e3b1-11e6-9d86-6a50ff545626.png)

**Comment by ApacheThunder** [comment]
I'll compile the original source and see if it's related to my changes.

**Comment by ApacheThunder** [comment]
Ok confirmed it was something I did. Though the change I've been maintaining only involed this edit:

Original (line 48 in sav6.cs, line 84 in sav7.cs): public override int MaxSpeciesID => Legal.MaxSpeciesID_6;

Modified:
Gen 6:
public override int MaxSpeciesID => 722;
Gen 7:
public override int MaxSpeciesID => 804;

I made this change to allow generating the "Egg" Pokémon that appears in the undefined species slot after Marshadow in Gen 7 and Volcanion in gen 6. Egg more or less worked in Gen 6.

It crashes in gen 7 though and now that I think about it perhaps I should retire this build of PkHeX. I use model swap to achieve Egg in battle now anyways. :P 

I'll close this issue since it's not the fault of the original code. Though I wonder why this suddenly caused QR genning option to be greyed out and disabled. Must have been one of the newer commits. It was a recent changed that broke it. But I don't really care enough to fix this as I'm going to retire this to a gen 6 only pkhex which I already have a backup copy to use for that. :P
